go_modules/lib/dependabot/go_modules/file_parser.rb

```diff
@@ -85,56 +95,63 @@ def required_packages
 
             stdout, stderr, status = Open3.capture3(command)
             handle_parser_error(path, stderr) unless status.success?
-            JSON.parse(stdout)["Require"] || []
-          end
+            T.let(JSON.parse(stdout)["Require"] || [], T::Array[T::Hash[String, T.untyped]])
```

Comment:
Is this necessary? Or is it also caught by the outer `T.let`?

Reply:
Good catch, doesn't look like it's necessary!